workspace/Dockerfile

```diff
@@ -106,15 +106,6 @@ RUN sudo -u ubuntu -i bash -l -c " \
 RUN wget -qO- https://toolbelt.heroku.com/install-ubuntu.sh | sh -e
 RUN sudo -u ubuntu -i heroku --help
 
-# MongoDB
-RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10 && \
-    echo "deb http://downloads-distro.mongodb.org/repo/ubuntu-upstart dist 10gen" > /etc/apt/sources.list.d/mongodb.list && \
-    apt-get update && \
-    apt-get install -y mongodb-org
```

Comment:
just remove this line. We can keep the others, which makes it easier for users to install mongo